Add katana CI to github workflows (#148)
lambdaclass/yet-another-swap: .github/workflows/README.md

# Testing GitHub workflows locally

## Using 'act' (`https://github.com/nektos/act`) to test GitHub actions locally

First download `act` tool

```bash
curl -s https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash
```

It is possible you may have to move it to the appropiate `/bin` folder:

```bash
mv ./bin/act /bin/act
```

Now to verify it is correctly installed:

```bash
act --version
```

> You will have to specify the 'workflows' folder with `-a` parameter:
> 
> ```bash
> act -a .github/workflows/
> ```
> 

Reviewer: ```suggestion
### How to use act

To run all the 'workflows':

```bash
act -a .github/workflows/
```

For MacOS users with M1 processor or newer versions:
```
act -a .github/workflows/ --container-architecture linux/amd64
```
```

And if you want to only run one of the github events you can specify it with the `-j` flag

```bash
act -a .github/workflows/ -j katana

> ```

Reviewer: Please add that for Mac users with M1 you need to use this flag:
```--container-architecture linux/amd64 ```
